mlir/utils/jenkins/Jenkinsfile

```diff
@@ -65,6 +65,7 @@ String dockerArgs() {
 }
 
 String dockerImage() {
+    // If this is being changed please change Dockerfile.ci-migraphx's base image as well
```

Comment:
If I read it right, now we have to set of images:
- rocm/mlir:rocm4.5-latest
- rocm/mlir.ci-migraphx:rocm4.5-latest

My questions are:
- Is the channel `rocm/mlir.ci-migraphx` created? I don't know if you start pushing to a non-existent channel, docker hub will create it.
- When will CI ever pull in the `rocm/mlir.ci-migraphx:rocm4.5-latest` images? I see we populate images but don't see a place that would consume those. Do you plan to work on that in follow-up PRs?

Reply:
@jerryyin,

I think it will be created, if I am not misataken and Jenkins user has authority. If not I ll reach out.

Yea the following PR would pull them. I would need to see the logs in the PR branch see if everything is in place once this PR is landed.

Reply:
( I ll look into it tomorrow once Im back)

Reply:
@manupak I don't know you are on vacation. Our team don't expect anyone on vacation to reply to github messages or emails. Otherwise thanks for the clarification and it sounds good.

Reply:
@jerryyin I think you might be right about push event not creating a repository. (I think the "channel" vs tag confused me :) ).

So yes I need some help to get a repo created then.

Reply:
(Sorry, I forgot to update my MS status)